### ISSUE TITLE
Code for §4.3.7 Node Array A*

### ISSUE THREAD

**Issue by legends2k** [idmillington/aicore#11]
Hi
Thanks for this book, it's a wealth of useful information. In the second edition of your book, you discuss about a simple A* implementation where all nodes are stored in an array, named _Node Array A*_. The book says

> The node array pathfinding implementation on the website uses…

However, I don't find the code here. Can you please commit this code if you have it lying around on one of your old hard drives? :)

**Comment by Favorlock** [comment]
I agree, it would be helpful if repo reflected what was expressed in the book.
